syz-ci/manager.go

```diff
@@ -954,7 +955,7 @@ func (mgr *Manager) uploadCoverStat(fuzzingMinutes int) error {
 	if err := mgr.uploadCoverJSONLToGCS(nil,
 		"/cover?jsonl=1&flush=1",
 		mgr.cfg.CoverPipelinePath,
-		curTime,
+		dateTimeSuffix(time.Now()),
```

Comment:
Do you ever supply any other suffixes? If not, maybe we can add a boolean flag telling uploadCoverJSONLToGCS() to add date/time?

Reply:
It was my first idea. The problem is how to mock the time.